generated/thrift/rpc.thrift

```diff
@@ -106,6 +110,48 @@ struct FetchBatchRawResult {
 	1: required list<FetchRawResult> elements
 }
 
+struct FetchTaggedRequest {
+	1: required IdxQuery query
+	2: required i64 rangeStart
+	3: required i64 rangeEnd
+	4: required bool fetchData
+	5: optional i64 limit
+	6: optional binary pageToken
+	7: optional TimeType rangeTimeType = TimeType.UNIX_SECONDS
+}
+
+struct IdxQuery {
+	1: required BooleanOperator operator
+	2: optional list<IdxTagFilter> filters
+	3: optional list<IdxQuery> subQueries
+}
+
+enum BooleanOperator {
+	AND_OPERATOR,
+}
+
+struct IdxTagFilter {
+	1: required binary tagName
+	2: required binary tagValueFilter
+	3: required bool negate
+	4: required bool regexp
+}
+
+struct FetchTaggedResult {
+	1: required list<FetchTaggedIDResult> elements
+	2: required bool hasMore
+	3: optional binary pageToken
```

Comment:
Might be simple to just make the semantics similar to fetch blocks metadata where pageToken is `nil` if no more results, otherwise if not `nil` then there are more?  The `hasMore` fields is kind of duplicative.

Reply:
I reworked this to simplify the initial implementation - no paging token. Input only has a limit and on the return side a bool to indicate if the results are exhaustive. Can add the paging once we implement that query side in the indexing layer.